debian/marquee/nginx/conf/whatwg.conf

```diff
@@ -0,0 +1,29 @@
+ssl_session_cache shared:SSL:10m;
+ssl_session_timeout 5m;
+
+ssl_protocols TLSv1 TLSv1.1 TLSv1.2;
+
+ssl_prefer_server_ciphers on;
+ssl_ciphers "ECDHE-RSA-AES128-GCM-SHA256 EECDH+ECDSA+AESGCM EECDH+aRSA+AESGCM EECDH+ECDSA+SHA384 EECDH+ECDSA+SHA256 EECDH+aRSA+SHA384 EECDH+aRSA+SHA256 EECDH+aRSA+RC4 EECDH EDH+aRSA RC4 !aNULL !eNULL !LOW !MD5 !EXP !PSK !SRP !DSS !MEDIUM !RC4 !CAMELLIA !3DES";
```

Comment:
I think we should do modern btw and disable TLSv1.

Reply:
ssl_ciphers "ECDHE-RSA-AES128-GCM-SHA256 EECDH+ECDSA+AESGCM EECDH+aRSA+AESGCM EECDH+ECDSA+SHA384 EECDH+ECDSA+SHA256 EECDH+aRSA+SHA384 EECDH+aRSA+SHA256 EECDH+aRSA+RC4 EECDH EDH+aRSA RC4 !aNULL !eNULL !LOW !MD5 !EXP !PSK !SRP !DSS !MEDIUM !RC4 !CAMELLIA !3DES"; 
> Can you add a comment about the source of list, since best practices will change?

The source of the list is that it’s what I manually set up for my own server based on information I had at hand at the time. So I can’t reconstruct the rationale for it.

> Maybe it's https://mozilla.github.io/server-side-tls/ssl-config-generator/?

Yeah, if we’re going to go about it systematically, I guess we should use that tool and replace that current `ssl_ciphers` with whatever we get back from that tool.

Reply:
> I think we should do modern btw and disable TLSv1

I agree but want to get an OK from @foolip before changing that, because I recall he said previously he wasn’t convinced we should drop it.

Reply:
> I think we should do modern btw and disable TLSv1.

To be clear, if we use “Modern” from https://mozilla.github.io/server-side-tls/ssl-config-generator/ it means TLSv1.1 will also be disabled—not just TLSv1